main/src/ca/uwaterloo/flix/runtime/shell/Shell.scala

```diff
@@ -312,9 +312,8 @@ class Shell(bootstrap: Bootstrap, options: Options) {
         // Cast to allow any subset of the primitive effects
```

Comment:
We should probably update the documentation here, and insert a reference to EntryPoints.